release-notes/5.0/preview/api-diff/preview2/Asp.Net/5.0-preview2_Microsoft.AspNetCore.Authentication.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft.AspNetCore.Authentication
+
+``` diff
+ namespace Microsoft.AspNetCore.Authentication {
+     public class AuthenticationSchemeProvider : IAuthenticationSchemeProvider {
++        public virtual bool TryAddScheme(AuthenticationScheme scheme);
+     }
+     public interface IAuthenticationSchemeProvider {
++        bool TryAddScheme(AuthenticationScheme scheme);
```

Comment:
Note that this diff does not distinguish default interface methods. I checked this one, and it's not a breaking change 👍 https://github.com/dotnet/aspnetcore/commit/a0eb923ab58193944f333880b2ab61ce2ca62725